docs/learning/78-nextjs-three-howler-dev-memory-explosion-diagnosis.md

```
@@ -0,0 +1,359 @@
+- 가설이 다 틀렸나? 그렇다고 보기엔 fix들 자체는 유의미하다 (실제 leak 위험 차단).
+- 다른 원인이 더 있나? 환경 상태를 의심해야 할 시점.
+
+---
+
+## 2차 진단 — 환경 상태를 보기 시작
+
+dev 서버 폭주는 코드만의 문제가 아닐 수 있다. **빌드 캐시·node_modules 같은 환경 상태**도 의심해야 한다.
+
+`frontend/.next` 디렉토리 크기 확인:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)